### ISSUE TITLE
Render toolbars with a different colour from the main window

### ISSUE THREAD

**Issue by tduck973564** [Bali10050/Lightly#21]
![image](https://github.com/user-attachments/assets/7c17832e-6718-4487-89dd-c28654ade4b9)

This area should (optionally) use the same colour as the titlebar. Looks weird without it. I may be able to look at this at some point.

This is how it's done in Breeze:
![image](https://github.com/user-attachments/assets/099817fd-4858-4866-835c-3d883d5530f9)

**Comment by Bali10050** [comment]
The color of the titlebar is definied by the colorscheme. You can change it any time you want.
![Screenshot_20240926_174054](https://github.com/user-attachments/assets/1e1c8172-5fbf-4c2e-928c-cfbd75353246)

This is how it looks when you use the color scheme that I provided with the install:

![Screenshot_20240926_174011](https://github.com/user-attachments/assets/3507d69e-bd10-442c-8a17-6dc9cc247c1f)

Also, the window decoration is now just a leftover, and I recommend using [klassy](https://github.com/paulmcauley/klassy) until I make my own

**Comment by tduck973564** [comment]
![image](https://github.com/user-attachments/assets/289a38cb-9df1-4b44-9ae1-7ebc8036de4d)

Nope. Still the same in Dolphin. It only works in QtQuick applications, because they're barely being touched in the first place.

**Comment by tduck973564** [comment]
> I have understanding problems, did you mean something like this?

Yes, but it should be the same as the titlebar (that could be a colour scheme thing). In any case it should be darker than the window colour.

**Comment by Bali10050** [comment]
I'll add this to the ideas list, I tried changing some values around in `lightlystyle.cpp`, but none of them seem to have an effect on that part, it might need some code that isn't in my skillset yet.

**Comment by christiantosta** [comment]
Brise, a fork of breeze set the menu to the decoration color. With Lightly and Vinyl you can do that by tunning the color scheme.

**Comment by DeltaCopy** [comment]
@tduck973564 

Made a few changes in the code (not pushed yet)
Now the toolbar colors seem to be darker and give the apps a more uniform look.

## Before look

### Dolphin

![dolphin_before](https://github.com/user-attachments/assets/bc306310-c5e9-405a-a217-10205b07b8f8)

### Kate

![kate_before](https://github.com/user-attachments/assets/91f5d06b-06e8-4aaf-b88d-7432a19cb40e)

## After look

### Dolphin

![dolphin_after](https://github.com/user-attachments/assets/1dfd18e5-bdf0-453c-8b5a-edd1c40594ec)

### Kate

![kate_after](https://github.com/user-attachments/assets/a334518d-5406-4980-b7f3-a063dc14cee2)

**Comment by DeltaCopy** [comment]
@Bali10050 please close this as it was already resolved in #42
